### DISCUSSION TITLE
bevallás befogadás visszaigazolása

### DISCUSSION THREAD

**Discussion by dgesler** [nav-gov-hu/eVAT#33] [Kérdések és válaszok - Gépi interfész]
Kérjük szíves tájékoztatásukat, hogy amennyiben az adózó a NAV által kiajánlott áfa analitikát bevallásként elfogadja és a külön applikáción jóváhagyja, milyen módon fog visszajelzést kapni a beküldés megtörténtéről és az adóhatóság általi fogadásról (jelenleg feladási igazolást és letöltési igazolást kapunk a cégkapun keresztül).

Köszönettel,
MOL Group

**Comment by NTCA-tax** [comment]
Az issue-ban megválaszoltam.
